docs/usage/configuration-options.md

```diff
@@ -1248,6 +1248,27 @@ This way the secret can be used in both the `org1` and `org2` organizations.
 
 For more information on how to use secrets for private packages, read [Private package support](./getting-started/private-packages.md).
 
+## env
+
+This option allows users to specify explicit environment variables and their corresponding values.
+
+<!-- prettier-ignore -->
+!!! note
+    This option is only available to users if the bot administrator has configured an [`allowedEnv`](./self-hosted-configuration.md#allowedEnv) list permitting the env names you wish to use.
```

Comment:
```suggestion
    The bot administrator must configure a list of allowed environment names in the  an [`allowedEnv`](./self-hosted-configuration.md#allowedEnv) config option, before users can use those allowed names in the `env` option.
```

How about something like this? I'm trying to explain:

- That bot admin must set allowed names in the `allowedEnv` list
- That users can only use names from the _allowed_ list in their `env` config option